package.json

```diff
@@ -45,5 +46,8 @@
     "jest": "^26.6.3",
     "prettier": "^2.2.1",
     "typescript": "^4.2.4"
+  },
```

Comment:
It is added, as "@types/yargs" on line 37

Reply:
Don't you need yargs itself added? 

Didn't you run `npm install yargs`?

Reply:
`@types/node` should be a dev depenency

Reply:
In case we haven't said so explicitly, dependencies should never be added manually to `package.json`. We should always run install commands, e.g. `npm i -D yargs @types/yargs`